supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -13,6 +143,39 @@ For documentation questions, contact rhel-docs@redhat.com.
 
 *See also*: xref:upgrade-rhel[upgrade], xref:in-place-upgrade[in-place upgrade]
 
+[discrete]
+[[commit]]
+==== image:images/yes.png[yes] commit (noun)
+*Description*: A release or image version of the operating system. Image Builder generates an OSTree commit for RHEL for Edge images. You can use these images to install or update RHEL on Edge servers.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
```

Comment:
Shouldn't this be `xref:ostree[OSTree]` (per the entry it links to)?

Same for a few other entries that link to it

Reply:
+1